content/en/docs/contributing/pull-requests.md

```diff
@@ -289,15 +289,15 @@ Figure 4 shows the steps to open a PR from your fork to
 flowchart LR
 subgraph first[ ]
 direction TB
-1[1. Go to opentelemetry.io repository] --> 2[2. Select New Pull Request]
-2 --> 3[3. Select compare across forks]
-3 --> 4[4. Select your fork from<br>head repository drop-down menu]
+1[1\. Go to opentelemetry.io repository] --> 2[2\. Select New Pull Request]
+2 --> 3[\.3 Select compare across forks]
+3 --> 4[\.4 Select your fork from<br>head repository drop-down menu]
 end
 subgraph second [ ]
 direction TB
-5[5. Select your branch from<br>the compare drop-down menu] --> 6[6. Select Create Pull Request]
-6 --> 7[7. Add a description<br>to your PR]
-7 --> 8[8. Select Create pull request]
+5[5\. Select your branch from<br>the compare drop-down menu] --> 6[6\. Select Create Pull Request]
+6 --> 7[\.7 Add a description<br>to your PR]
+7 --> 8[\.8 Select Create pull request]
```

Comment:
You've inverted the order of the period and the number, which results in an error, see https://deploy-preview-6253--opentelemetry.netlify.app/docs/contributing/pull-requests/#open-a-pr:

> <img width="575" alt="image" src="https://github.com/user-attachments/assets/ded4c319-6c98-40d0-b50b-dd19ab675edd" />

Reply:
Sorry, I was mistaken and caused other errors. I fixed this commit. Please check it.

https://github.com/open-telemetry/opentelemetry.io/pull/6253/commits/90f430ff6adb59226eaa37114461b39b497eda30